model/metrics/system-metrics.yaml

```diff
@@ -340,3 +340,16 @@ groups:
     instrument: updowncounter
     unit: "By"
     attributes: []
+
+  - id: metric.system.linux.memory.slab.usage
+    type: metric
+    metric_name: system.linux.memory.slab.usage
+    stability: experimental
+    brief: "Reports the memory used by the Linux kernel for managing caches of frequently used objects."
+    note: |
+      The sum over all `system.linux.memory.slab.usage` state values SHOULD be equal to the total slab memory available on the system.
+      Note that the total slab memory is not constant and may vary over time.
```

Comment:
Would be super helpful if there was any links to any kind of docs that mention more about that.

Reply:
Definitely, added a couple of links in https://github.com/open-telemetry/semantic-conventions/pull/1078/commits/6b01a6fbcd79fed011cb7eae3be9f474dbe7e2c9